caiman/base/timeseries.py

```diff
@@ -253,7 +253,6 @@ def concatenate(*args, **kwargs):
     """
     # todo: todocument return
 
-    obj = []
```

Comment:
Initially having this be a list didn't serve a point and resulted in it changing types in the function body. Removing this makes the type consistent across the function.